template/{{cookiecutter.module_name}}/1.0/{{cookiecutter.module_name}}.smk

```diff
@@ -11,10 +11,27 @@
 
 ##### SETUP #####
 
-
 # Import package with useful functions for developing analysis modules
 import oncopipe as op
 
+# Check that the oncopipe dependency is up-to-date. Add all the following lines to any module that uses new features in oncopipe
```

Comment:
This will be redundant once you implement the handling of the version checker by separate function as discussed at the last meeting, so I would think this update is not needed here.

Reply:
@rdmorin Sir, should I leave it blank ,or fill it with the code until we change it.